src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,261 @@
+                    break;
+            }
+        } while (!visualUserMenu.validateIntegerSize(ListMenu.getAuthorizationMenu().size(), action));
+    }
+
+    private void registrationUserMenu() {
+        String firstName = "";
+        String lastName = "";
+        boolean repeat = true;
+        do {
+            firstName = visualUserMenu.getValidInputFromUser("Enter your first name: ", InputType.STRING);
+            if (firstName.contains(" ") || firstName.isEmpty()) {
+                System.out.println(ERROR_INVALID_NAME_FORMAT);
+                continue;
+            }
+            lastName = visualUserMenu.getValidInputFromUser("Enter your last name: ", InputType.STRING);
+            if (lastName.contains(" ") || lastName.isEmpty()) {
+                System.out.println(ERROR_INVALID_NAME_FORMAT);
+                continue;
+            }
+            repeat = false;
+        } while (repeat);
+        userController.loginUser(firstName, lastName);
+        System.out.println(firstName + ", You have successfully logged in!\n\n\t Welcome to our site!\n");
+    }
+
+    public void serviceMenu() {
+        while (true) {
+            VisualUserMenu.printListInConsole(ListMenu.getServiceHeader(), ListMenu.getServiceMenu());
+            Integer action;
+            action = Integer.parseInt(visualUserMenu.getValidInputFromUser("Choose action: ", InputType.INTEGER));
+            switch (action) {
+                case (1):
+                    VisualUserMenu.outputSplitLine();
+                    findHotelByNameMenu();
+                    break;
+                case (2):
+                    VisualUserMenu.outputSplitLine();
+                    VisualUserMenu.printListInConsole(null, findHotelByCityMenu());
+                    break;
+                case (3):
+                    VisualUserMenu.outputSplitLine();
+                    findRoomMenu();
+                    break;
+                case (4):
+                    VisualUserMenu.outputSplitLine();
+                    getAllUserReservationsMenu();
+                    break;
+                case (5):
+                    VisualUserMenu.outputSplitLine();
+                    userController.logoutUser();
+                    return;
+                case (6):
+                    VisualUserMenu.outputSplitLine();
+                    System.out.println("\tThank you for using our service.");
+                    System.exit(0);
+                    break;
+                default:
+                    VisualUserMenu.outputSplitLine();
+                    System.out.println(ERROR_INCORRECT_MENU_ITEM_SELECTED);
+
+            }
+        }
+    }
+
+    private List<Hotel> findHotelByNameMenu() {
+        while (true) {
+            String hotelName = visualUserMenu.getValidInputFromUser("Enter hotel name (enter \"0\" to return back to "
+                + "menu): ", InputType.STRING);
+            if (hotelName.equals("0")) {
+                return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1168)